### PR TITLE
Pavel/chips

### PR DESCRIPTION
**Pull request by galloween**
Renamed Badge to Chip, now uses mat-chip internally (at the moment does not add any additional fucntionality)

### CODE REVIEW

File: projects/ui-framework/src/lib/buttons-indicators/chips/chip/chip.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(ChipComponent);
+        component = fixture.componentInstance;
+      });
+  }));
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/buttons-indicators/chips/chip/chip.component.spec.ts]: we can remove this test

File: projects/ui-framework/src/lib/buttons-indicators/chips/chip/chip.component.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Component, Input, HostBinding } from '@angular/core';
+import { ChipType } from '../chips.enum';
+
+@Component({
+  selector: 'b-chip, [b-chip]',
+  template: `
+    <mat-basic-chip [selectable]="false" [disableRipple]="true">
+      <ng-content></ng-content>
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/buttons-indicators/chips/chip/chip.component.ts]: what's the ng-content for?

Comment (galloween) [reply projects/ui-framework/src/lib/buttons-indicators/chips/chip/chip.component.ts]: it allows to be used like this <b-chip>hello</b-chip> or <p b-chip>item</p>
material chip also behaves like that (you can pass [value] binding, or <mat-chip>text</mat-chip>)
also maybe we need to pass avatar or icon or button there in the future.
just for flexibility, basically. i can remove it, if we want stricter api

File: projects/ui-framework/src/lib/buttons-indicators/chips/chip/chip.component.spec.ts (hunk 1 above)
Comment (galloween) [reply projects/ui-framework/src/lib/buttons-indicators/chips/chip/chip.component.spec.ts]: yes, it was in button test I used as reference. but its unnecessary of course. any other test will fail if component is not created.

**Pull request merged**